src/pages/static/assets/global.css

```
@@ -51,18 +51,52 @@ html {
 }
 
 /* EDITOR CONTENT STYLES */
-.toastui-editor-main * {
+.toastui-editor-contents * {
   font-family: poppins, sans-serif !important;
 }
-.toastui-editor-main h1,
-.toastui-editor-main h2,
-.toastui-editor-main h3 {
+.toastui-editor-contents h1,
+.toastui-editor-contents h2,
+.toastui-editor-contents h3,
+.toastui-editor-contents h4,
+.toastui-editor-contents h5,
+.toastui-editor-contents h6 {
   border-bottom: none !important;
   padding-bottom: 0 !important;
-  margin-bottom: 0.5em !important;
-  margin-top: 1em !important;
+  margin: 24px 0 16px !important;
+  font-weight: 600;
+  line-height: 1.25;
+  scroll-margin-top: 80px;
+}
+.toastui-editor-contents h1,
+.toastui-editor-contents h2 {
+  font-weight: 500 !important;
+}
+.toastui-editor-contents h1 {
+  margin: 0.6em 0 !important;
+  font-size: 1.7em !important;
+}
+.toastui-editor-contents h4 {
+  font-size: 16px !important;
+}
```

Just as an example to buttress my previous point
```suggestion
:where(.toastui-editor-contents h4) {
  font-size: 16px;
}
```
Note that I removed the `!important` This would help in places where you have to sacrifice theme tokens for the sake of using `!important`